runtime/gc_base/IndexableObjectAllocationModel.cpp

```diff
@@ -58,7 +59,10 @@ MM_IndexableObjectAllocationModel::initializeAllocateDescription(MM_EnvironmentB
 #endif /* defined (J9VM_GC_MODRON_COMPACTION) || defined (J9VM_GC_GENERATIONAL) */
 	spineBytes = extensions->objectModel.adjustSizeInBytes(spineBytes);
 
+	bool isVirtualLargeObjectHeapEnabled = extensions->indexableObjectModel.isVirtualLargeObjectHeapEnabled();
+
 	/* determine size of layout overhead (additional to spine bytes) and finalize allocation description */
+	/* it is used to track amount of array bytes not included to spine and set final total number of bytes allocated for array */
```

Comment:
Start commments with a capital, and try to break into multiple (< 80 char)  lines, and add a period at the end.